theseus_gui/src/pages/project/Index.vue

```diff
@@ -236,12 +241,19 @@ watch(
 
 dayjs.extend(relativeTime)
 
-async function install() {
+async function install(version) {
   if (data.value.project_type === 'modpack') {
-    let id = await pack_install(versions.value[0].id)
-
-    let router = useRouter()
-    await router.push({ path: `/instance/${encodeURIComponent(id)}` })
+    const packs = Object.values(await list())
+    console.log(packs)
+    if (
+      packs.length === 0 ||
+      !packs.map((value) => value.metadata).find((pack) => pack.linked_project_id === data.value.id)
+    ) {
+      let id = await pack_install(version)
```

Comment:
Unhandled Promise Rejection: invalid args `versionId` for command `pack_install_version_id`: command pack_install_version_id missing required key versionId

this causes this error- looks like the bindings were updated, you'l probably have to rename the helper method to pass `versionId`

Reply:
this is from clicking the install button on the side panel